mantidproject/mantid
# In ALC Rename Auto Checkbox And Update Error Messages To Be More Meaningful

**Description of work.**
Based off feedback during beta testing these small changes have been applied to the ALC Data Loading interface. Checkbox previously called **Auto** has been renamed to be more clear as it's functionality has been altered since the previous release. It is now called **Auto Find Current Run**. Additionally when the runs box is invalid the error message should now be more clear and addresses a common misconception that you can enter runs in reverse order (which is not allowed). Also updated release notes for Muon so check they are okay too.

**Report to:** Stephen Cottrell

**To test:**

1. Open the ALC Interface from **Interfaces** -> **Muon** -> **ALC**
2. Check visually the check box next to runs no longer says **Auto** and is now **Auto Find Current Run**
3. Check the tooltip works for this checkbox and that it makes sense
4. Now try to enter some invalid runs. As an example :
   1. Try loading with no runs entered.
   2. Try loading runs which don't exist
   3. Try entering an invalid expression such as a range that decreases instead of increases
   4. Anything else you can think of that might cause runs to be invalid
5. Want to make sure the error message is more meaningful in all cases
6. Double check the docs to make sure they reflect the updates here (i.e. the screenshot of the interface and data loading description), and that the muon release notes make sense


Fixes #29495 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
